.circleci/config.yml

```diff
@@ -568,7 +568,7 @@ workflows:
       - integration_test:
           name: redshift-s3-itest-old-pandas
           extras: '[redshift-binary,itest]'
-          python_version: "3.9"
+          python_version: "3.8"
```

Comment:
why is this being changed to 3.8?

Reply:
Ah I'm guessing it's because this test specifically is targeting compatibility with an old version of pandas that maybe requires an older python version? the rest of these have been bumped to 3.9 as expected.

Reply:
you got it!

Reply:
Tried to find a way to get around this by allowing it to build the old version of pandas in 3.9 but kept creating compounding errors